patch_parser.py

```diff
@@ -650,7 +651,7 @@ def main():
     map_comments_to_gerrit(patchset)
     for patch in patchset.patches:
         for comment in patch.comments:
-            print('At ' + str(comment.raw_line) + ': ' + str(comment.file) + ':' + str(comment.line) + ':\n' + comment.message)
+            logging.info('At %s: %s: %s:\n %s', str(comment.raw_line), str(comment.file), str(comment.line), comment.message)
```

Comment:
ditto %d for raw_line

comment.line might be None if there's a logic bug, so perheaps we leave it as %s and drop the cast